manager/controlapi/server_test.go

```diff
@@ -39,13 +39,14 @@ type testServer struct {
 	Store  *store.MemoryStore
 
 	grpcServer *grpc.Server
-	clientConn *grpc.ClientConn
+
+	tempUnixSocket string
 }
 
 func (ts *testServer) Stop() {
-	_ = ts.clientConn.Close()
```

Comment:
We should close the client connection in this `Stop` function. I see that the connection is not being stored to the struct, but that should be fixed instead of removing the `Close`.